### ISSUE TITLE
[MU4 Issue] Key presses in file picker do not register, and are passed through to the score

### ISSUE THREAD

**Issue by llui85** [musescore/MuseScore#11193]
## **Describe the bug**
When a key is pressed in the file picker (such as the first letter of a file, arrow keys, or enter), it does not trigger an action in the file picker; instead, the action for that key occurs in the score underneath.

## **To Reproduce**
1. Open the file picker.
2. Try and use type-ahead search, or open a folder using the enter key.
3. Observe that your actions affect the score below the file picker.

## **Screen Recording**

I am pressing the first letter of each file in the directory (a, b, d), then the arrow keys.

https://user-images.githubusercontent.com/47158068/163363694-e5b2c7fb-a601-4281-9f57-dae3a0d16abc.mov

## **Platform information**
```
Linux 5.10.0-12-amd64 #1 SMP Debian 5.10.103-1 (2022-03-07) x86_64 GNU/Linux
```

**Comment by MarcSabatella** [comment]
Aha!  This is indeed what I was seeing what I mentioned the default button not activating in https://github.com/musescore/MuseScore/issues/10460.  If I have a measure selected in my current score, then go to File / Open and try to open a new file, then indeed, hitting Enter after selecting a file in the file dialog adds system break to my already-open score.

**Comment by Eism** [comment]
Related to #10836

**Comment by DmitryArefiev** [comment]
Fixed in #10836

@llui85 You recheck in next master. It works on my side fine
